community/samples/serving/helloworld-java-quarkus/README.md

```diff
@@ -0,0 +1,452 @@
+---
+title: "Hello World - Java (Quarkus)"
+linkTitle: "Java"
+weight: 1
+type: "docs"
+---
+
+{{% readfile file="README.md" relative="true" markdown="true" %}}
+
+A simple [JAX-RS REST API](https://github.com/jax-rs) application that is written in Java and uses [QuarkusIO](https://quarkus.io/).
+
+It uses an ENV variable `MESSAGE_PREFIX`(defaults to value `Hello`) and prints a message of format `${MESSAGE_PREFIX} Knative World!`.
+
+## Before you begin
+
+You must meet the following requirements to run this sample:
```

Comment:
Please see open comments/suggestions from first review. 
- Is both Knative Build and Docker required?
- The suggested edits make the list parallel and highlight options between docker or Knative Build